configyaml/config/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .base import ConfigBase
-from .base_list import ConfigBaseList
-from .base_dict import ConfigBaseDict
-from .base_wildcard_dict import ConfigBaseWildcardDict
-from .base_choice import ConfigBaseChoice
+from .base import AbstractNode
+from .list import NodeList
+from .dict import NodeDict
+from .wildcard_dict import WildcardDict
+from .choice import NodeChoice
```

Comment:
I'm good with whatever you decide. I have a bad habit of over-architecting from the start and wasting energy there. We can always change things if we make a life-threatening mistake.